### DISCUSSION TITLE
On Safari parts of the 3D Model stay static

### DISCUSSION THREAD

**Discussion by Bersaelor** [google/model-viewer#3478] [Q&A]
### Description

Using modelviewer on Chrome works fine, but in latest Safari only parts of the model change.
Try the [this example](https://grafix-eyewear.com/modell/bloxx-9/) on Safari vs Chrome:
(The same happens for all our models)

#### Live Demo

https://grafix-eyewear.com/modell/bloxx-9/

```
 <model-viewer id="mv" style="width: 100%; height: 400px;" src="https://render.looc.io/grafix/bloxx/bloxx9/1643021268526/bloxx9-1643021268526.glb" ios-src="https://render.looc.io/grafix/bloxx/bloxx9/1643021268347/bloxx9-1643021268347.usdz" alt="3D model GRAFIX eyewear" ar="true" ar-modes="webxr scene-viewer quick-look" ar-status="not-presenting" environment-image="neutral" loading="eager" camera-target="auto -20mm -100mm" max-field-of-view="45deg" auto-rotate="true" camera-controls="true" exposure="0.42" shadow-intensity="0.6" shadow-softness="0.5"></model-viewer>
```

### Version

- model-viewer: v?.?.?

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [ ] Windows

**Comment by samaneh-kazemi** [comment]
Hi @Bersaelor, thanks for reaching out. Can you please share two videos of what you see? I'm not able to repro this, but I'm not also quite sure what is the expectation.

However, based on what you're sharing, this sounds like a Safari bug, not a model-viewer.

**Comment by Bersaelor** [comment]
Here's a video of the problem:

https://user-images.githubusercontent.com/4517582/168665513-0c96705f-fa5b-4cb6-9f87-cdf7fed8fccf.mov

Tested on desktop Safari 12.3.1 as well as mobile iOS 15.4.1.

The bug is definitely based on some difference of WebGL implementation between Chrome and Safari, but I don't know enough about why this happens to know whether it's due to an intentional change in Safari's WebGL implementation or a bug on their part.

**Comment by elalish** [comment]
Wow, yeah that's a bad Safari bug. Firefox and Chrome are working at least. Please file a bug on WebKit. They've been fixing several WebGL things lately; you may want to check the latest dev releases.

**Comment by elalish** [comment]
Sounds like this is fixed in Safari 15.5: https://github.com/mrdoob/three.js/issues/23730

Please let us know if you can still repro there or not.

**Comment by elalish** [comment]
Verified in #3327
